ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -726,7 +722,7 @@
 				6FAD019AC5D030B60C288001 /* [CP] Check Pods Manifest.lock */,
 				C4AE08FE2AC43A4800E13758 /* Sources */,
 				C4AE09002AC43A4800E13758 /* Resources */,
-				CC326BE194B68508452D7DF2 /* Frameworks */,
+				3F83D1AAB2844DCF34EE511C /* Frameworks */,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The Frameworks reference has been changed. If this was intentional and the new reference is correct, then it's fine. However, if this change was not intended, it could potentially lead to build or runtime issues.

```diff
- CC326BE194B68508452D7DF2 /* Frameworks */,
+ 3F83D1AAB2844DCF34EE511C /* Frameworks */,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->